Dockerfile

```diff
@@ -9,15 +9,17 @@
 #
 ###
 
-FROM alpine:3.2
+FROM alpine:latest
```

Comment:
I can agree on using a later alpine version, but `:latest` is not recommended. Could you change this to 3.5? That at least reflects the actual used version ;)